plugins/the_rules.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+from errbot import BotPlugin, re_botcmd
+
+
+class The_rules(BotPlugin):
+    """
+    List the bot rules
+    """
+
+    RULES = [
+        "A robot may not harm humanity, or, by inaction, allow humanity to "
```

Comment:
Not applying here, since applied in other PR, will merge that first and then this, so there are less conflicts :+1: